docs/source/topics/proc_accessing-the-internal-openshift-registry.adoc

```diff
@@ -0,0 +1,92 @@
+[id="accessing-the-internal-openshift-registry_{context}"]
+= Accessing the internal OpenShift registry
+
+The OpenShift cluster running in the {prod} virtual machine includes an internal container image registry by default.
+This internal container image registry can be used as a publication target for locally developed container images.
+To access the internal OpenShift registry, follow these steps.
+
+.Prerequisites
+
+* A running {prod} virtual machine.
+For more information, see link:{crc-gsg-url}#starting-the-virtual-machine_gsg[Starting the virtual machine].
+* A working [command]`oc` command.
+For more information, see link:{crc-gsg-url}#accessing-the-openshift-cluster-with-oc_gsg[Accessing the OpenShift cluster with `oc`].
+* For Docker, add `default-route-openshift-image-registry.apps-crc.testing` as an insecure registry.
+For more information, see link:https://docs.docker.com/registry/insecure/[the Docker documentation].
+
+.Procedure
+
+. Get the route to the internal OpenShift registry:
++
+[subs="+quotes,attributes"]
+----
+$ oc get route -n openshift-image-registry
+----
+
+. Check which user is logged in to the cluster:
++
+[subs="+quotes,attributes"]
+----
+$ oc whoami
+----
++
+[NOTE]
+====
+For demonstration purposes, the current user is assumed to be `kube:admin`.
```

Comment:
would that work as `developer`? or do we need elevated privileges?

Reply:
Technically, I believe it can, but the `oc` commands would need to be run with `--as system:admin`, which is essentially the same thing if I understand it correctly.

This does make me wonder if we should simply advocate for using `kube:admin` in the docs without suggesting otherwise, though. That would be clearer for the reader in my opinion.

Reply:
Seems I was incorrect, this procedure will work with the `developer` user without elevated privileges. :)